ui/app/models/gcp/config.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+export default class GcpConfig extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+
+  /* GCP creds, mutually exclusive with WIF fields */
+  @attr({
+    label: 'Config TTL',
+    editType: 'ttl',
+    helperTextDisabled: 'The TTL of generated tokens. Defaults to 1 hour.',
```

Comment:
Often we spell out `time-to-live` in parenthesis after `TTL` (though we're not consistent!) I think it's nice to include, but will defer to you. 

https://github.com/hashicorp/vault/blob/main/ui/app/models/ldap/role.js#L107

Reply:
Usually the explanation of the ttl is `helperTextEnabled` and the `helperTextDisabled` is the default (since it's unset). Is there a reason for deviating from that pattern here?

Reply:
Good call out. I'll amend.

Reply:
Update: confirmed with the API, there is no actual default values for ttl or max_ttl. Thus, I'll keep the `helperTextDisabled` so the user understands without toggling what the value is used for, but remove any mention of defaults.

Reply:
Hmm by "no default" do you mean `0` because that means it's not **set**, not that there's no ttl. Also looking at the docs, the subtext is a little different here https://developer.hashicorp.com/vault/api-docs/secret/gcp#ttl

You can ask a backend engineer or make a request to `sys/mounts/internal` and get what the system default is